### PR TITLE
Ensure lib/ directory is loaded in all envs

### PR DESCRIPTION
**Pull request by mjgiarlo**

### CODE REVIEW

File: config/environments/development.rb [hunk 1]
```diff
@@ -8,8 +8,8 @@
   # since you don't have to restart the web server when you make code changes.
   config.cache_classes = false
 
-  # Do not eager load code on boot.
-  config.eager_load = false
+  # Eager load code on boot to find production problems in dev
+  config.eager_load = true
```
Comment (jcoyne) [inline config/environments/development.rb]: I wouldn't do this as it departs from the rails conventions.

File: config/application.rb [hunk 2]
```diff
@@ -15,7 +15,8 @@ class Application < Rails::Application
       # Application configuration should go into files in config/initializers
       # -- all .rb files in that directory are automatically loaded.
 
-      config.autoload_paths += %W[#{config.root}/lib]
+      config.autoload_paths << Rails.root.join('lib')
```
Comment (jcoyne) [inline config/application.rb]: You can drop this line.

**Comment by jcoyne** [comment]
Here's how a rails maintainer would do it: https://github.com/rails/rails/issues/13142#issuecomment-275549669

**Comment by mjgiarlo** [comment]
@jcoyne 	ready for another look. I'd rather make this change than drop `lib/` into `app/`. Follow-up work could create `app/services/` (and other needed `app/` subdirs) and moving/renaming everything out of `lib/was/`.

**Pull request merged**